### PR TITLE
FHAC-1306 Adding WildFly server config files for quick setup

### PR DESCRIPTION
**Pull request by sovanncgi**

### CODE REVIEW

**Comment by sovanncgi** [comment]
@knguyenAMDEX please review

File: Product/Install/WildFly/templates/standalone.xml [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+                </authorization>
+            </security-realm>
+            <security-realm name="ApplicationRealm">
+                <server-identities>
+                    <ssl>
+                        <keystore path="modules/org/connectopensource/configuration/main/gateway.jks" relative-to="jboss.home.dir" keystore-password="changeit" alias="gateway"/>
+                    </ssl>
+                </server-identities>
+                <authentication>
+                    <truststore path="modules/org/connectopensource/configuration/main/cacerts.jks" relative-to="jboss.home.dir" keystore-password="changeit"/>
```
Comment (mhpnguyen) [inline Product/Install/WildFly/templates/standalone.xml]: it should be "modules/system/layers/base/org/connectopensource/configuration/main/gateway.jks"

File: Product/Install/WildFly/templates/standalone.xml [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+                    <properties path="mgmt-users.properties" relative-to="jboss.server.config.dir"/>
+                </authentication>
+                <authorization map-groups-to-roles="false">
+                    <properties path="mgmt-groups.properties" relative-to="jboss.server.config.dir"/>
+                </authorization>
+            </security-realm>
+            <security-realm name="ApplicationRealm">
+                <server-identities>
+                    <ssl>
+                        <keystore path="modules/org/connectopensource/configuration/main/gateway.jks" relative-to="jboss.home.dir" keystore-password="changeit" alias="gateway"/>
```
Comment (mhpnguyen) [inline Product/Install/WildFly/templates/standalone.xml]: similar like comments below

File: Product/Install/WildFly/templates/standalone.conf [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+# Uncomment this in order to be able to run WildFly on FreeBSD
+# when you get "epoll_create function not implemented" message in dmesg output
+#JAVA_OPTS="$JAVA_OPTS -Djava.nio.channels.spi.SelectorProvider=sun.nio.ch.PollSelectorProvider"
+
+# Increase memory allocation
+JAVA_OPTS="-Xmx5000m -XX:MaxPermSize=1024m -XX:PermSize=1024m"
+ 
+# configuration directory
+JAVA_OPTS="$JAVA_OPTS -Dnhinc.properties.dir=$JBOSS_HOME/modules/org/connectopensource/configuration/main/"
```
Comment (mhpnguyen) [inline Product/Install/WildFly/templates/standalone.conf]: missing "modules/system/layers/base/org/connectopensource/configuration/main"

File: Product/Install/WildFly/templates/standalone.conf [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+ 
+# keystore and truststore
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.keyStore=$JBOSS_HOME/modules/system/layers/base/org/connectopensource/configuration/main/gateway.jks"
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.keyStorePassword=changeit"
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.keyStoreType=JKS"
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStore=$JBOSS_HOME/modules/system/layers/base/org/connectopensource/configuration/main/cacerts.jks"
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStorePassword=changeit"
+JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStoreType=JKS"
+JAVA_OPTS="$JAVA_OPTS -DCLIENT_KEY_ALIAS=gateway"
```
Comment (mhpnguyen) [inline Product/Install/WildFly/templates/standalone.conf:85]: consider to add Dcom.sun.xml.ws.transport.http.HttpAdapter.dump=true and Dcom.sun.xml.ws.transport.http.client.HttpTransportPipe.dump=true" to dump out request and response in server log

Comment (sovanncgi) [reply Product/Install/WildFly/templates/standalone.conf:85]: I considered this but it's not standard WildFly configuration and not required for CONNECT

**Comment by knguyenAMDEX** [comment]
Reviewed and endorse for merging

**Pull request merged**